libs/designsystem/tab-navigation/src/tab-navigation/tab-navigation.component.scss

```diff
@@ -1,11 +1,35 @@
 @use '@kirbydesign/core/src/scss/utils';
 
-$tab-item-text-max-width: 100px;
 $bottom-border-height: 1px;
 $border-color-standard: utils.get-color('medium');
-$border-color-selected: utils.get-color('dark');
 $divider-max-width-breakpoint: utils.$page-content-max-width + (utils.size('s') * 2);
 
+:host {
+  display: block;
+  position: relative;
+
+  @include utils.media('<medium') {
+    margin-inline: calc(-1 * var(--padding-start)) calc(-1 * var(--padding-end));
+  }
+
+  // Divider
+  &::before {
+    content: '';
+    background-color: $border-color-standard;
+    position: absolute;
+    height: $bottom-border-height;
+    bottom: 0;
+    z-index: 1;
```

Comment:
This value can also be set by using `utils.z('default')`

Reply:
@mark-drastrup Good catch! 👌🏻